homeassistant/components/mass/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for Music Assistant Component."""
+
+import logging
+
+DOMAIN = "mass"
```

Comment:
let's make the domain `music_assistant`

Reply:
Hah, I already discussed this with both Marcel and Jozef :P

Reply:
Working on it as we speak